UNRELEASED.md

```diff
@@ -2,4 +2,12 @@
 
 - bpk-foundations-web:
     - Update banner alert tokens to use semantic colour tokens
-    - Deprecate `$bpk-banner-alert-event-color` token
\ No newline at end of file
+    - Deprecate `$bpk-banner-alert-event-color` token
```

Comment:
Might be worth checking with @anambl about this as it looks like this change may have already been released but changelog not updated

Reply:
Yea, it looks to be released. But the changelog entry is missing. 
I'll take care of that!

Reply:
Nope, wasn't released! I'd forgot to log out of artifactory so the release didn't work, but the CI still ran 🙂